modules/README.md

```diff
@@ -66,6 +66,15 @@ interface ISafeProtocolHooks {
 }
 ```
 
+### Parameter `executionMeta` value
+
+| Execution Type      | Value                                                                                                                                       |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| Multisignature Flow | Encoded data of the transaction i.e. `abi.encode(to, value, data,gas, baseGas, gasPrice, gasToken, refundReceiver, signatures, msgSender )` |
```

Comment:
There's a missing space after `data,` and a redundant one after msgSender

Reply:
It's a little bit vague what "encoded data of the transaction` means. Where do the values come from? Can they differ across accounts? How does it work with Safe account modules?